### PR TITLE
explain about the fork

### PR DESCRIPTION
**Pull request by ziirish**
Let the community know about the fork and give some context.

cc. @j5awry, @SteadBytes

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -21,6 +21,22 @@ Flask RestPlus
    :alt: Join the chat at https://gitter.im/noirbizarre/flask-restplus
    :target: https://gitter.im/noirbizarre/flask-restplus?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
 
+**IMPORTANT NOTICE**: The community has decided to fork the project due to lack
+of answers from the original author. We have been discussing this eventuality
+for almost [a year](https://github.com/noirbizarre/flask-restplus/issues/593).
+
+Things evolved a bit since that discussion and a few of us have been granted
+maintainers access to the github project, but we've been lacking rights on the
+Pypi project. Hence we haven't been able to release any new version.
```
Comment (SteadBytes) [inline README.rst]: Apologies for nitpicking, but a capital letter is missing: `Pypi -> PyPi` :sweat_smile:

**Comment by SteadBytes** [comment]
Thanks for putting this together @ziirish this is great. I'm just wondering if it might be beneficial to re-arrange slightly and *start* with the fork notice. This makes it immediately obvious that this project should be considered unmaintained and that the new fork is where ongoing development will take place. The explanation of why could then follow :) Something like this perhaps?

**IMPORTANT NOTICE**:

This project has been **forked** to [flask-restx](https://github.com/python-restx/flask-restx) and will be maintained by by the [python-restx](https://github.com/python-restx) organization. **Flask-RESTPlus should be considered unmaintained**.

The community has decided to fork the project due to lack of response from the original author @noirbizarre. We have been discussing this eventuality for almost [a year](https://github.com/noirbizarre/flask-restplus/issues/593).

Things evolved a bit since that discussion and a few of us have been granted maintainers access to the github project, but only the original author has access rights on the PyPi project. As such, we been unable to make any actual releases. To prevent this project from dying out, we have forked it to continue development and to support our users.

**Comment by ziirish** [comment]
Thanks for the review.

I'll update the README accordingly to your version which I prefer over mine :)

**Comment by SteadBytes** [comment]
Awesome thank you for taking the feedback on board! 😊

Would you mind fixing up those extra couple of commits into the first one? Just to keep the history cleaner, then we should merge! 🎉

**Comment by ziirish** [comment]
done

**Comment by rogeriolino** [comment]
Congrats guys. These project must (still) be maintaned.

But, what about the new name?

**Comment by ziirish** [comment]
@rogeriolino I'm not sure to understand your question.

We have added the links to the fork in the README and I opened and pinned an issue with all the informations: #770 

What would you like to know?

**Comment by rogeriolino** [comment]
@ziirish is about the org/project suffix `restx`.

I was following the #743 issue, where forks naming was suggested, and I dont read about the these new name. 

Just for curiosity.

Regards

**Comment by ziirish** [comment]
So you want to know why we chose `restx`?

I'm thinking of adding the reason in flask-restx' README at some point.
The idea comes from a discussion on the gitter channel. It's a proposal of @apryor6.
> `flask-restx` as an 'x' is just like a '+', with a twist :)

Most of us liked the idea, hence the new name.

The discussion is [here](https://gitter.im/noirbizarre/flask-restplus?at=5e14d8d309534142ca27fecf).

**Pull request merged**